test_scripts/Defects/7_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua

```diff
@@ -0,0 +1,137 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
should be added to 7_0 defects test_set?

Reply:
Good catch, fixed